composer.json

```
@@ -16,30 +16,30 @@
     ],
     "require": {
         "php": "^7.1",
-        "symfony/framework-bundle": "^3.4 || ^4.1",
+        "doctrine/common": "^2.0",
         "symfony/browser-kit": "^3.4 || ^4.1",
-        "doctrine/common": "^2.0"
+        "symfony/framework-bundle": "^3.4 || ^4.1"
     },
     "require-dev": {
-        "symfony/symfony": "^3.4|^4.1",
-        "symfony/phpunit-bridge": "^3.4|^4.1",
-        "doctrine/orm": "^2.6",
-        "symfony/monolog-bundle": "^3.1",
-        "doctrine/doctrine-fixtures-bundle": "^3.0.2",
+        "doctrine/data-fixtures": "^1.3",
         "doctrine/doctrine-bundle": "^1.8",
+        "doctrine/doctrine-fixtures-bundle": "^3.0.2",
+        "doctrine/orm": "^2.6",
+        "jackalope/jackalope-doctrine-dbal": "^1.3",
         "phpunit/phpunit": "^6.1 || ^7.0",
-        "twig/twig": "^2.0",
+        "symfony/monolog-bundle": "^3.1",
+        "symfony/phpunit-bridge": "^3.4 || ^4.1",
+        "symfony/symfony": "^3.4 || ^4.1",
         "theofidry/alice-data-fixtures": "^1.0.1",
-        "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "doctrine/data-fixtures": "^1.3"
+        "twig/twig": "^2.0"
     },
     "suggest": {
```

We need to commit the actual option:
```suggestion
    "config": {
        "sort-packages": true
    },
    "suggest": {
```